Kinvey/Kinvey/Error.swift

```diff
@@ -21,22 +21,22 @@ public enum Error: Swift.Error {
     case objectIdMissing
     
     /// Error when a method is not allowed, usually when you are using a Data Link Connector (DLC).
-    case methodNotAllowed(debug: String, description: String)
+    case methodNotAllowed(httpResponse: HTTPURLResponse?, data: Data?, debug: String, description: String)
```

Comment:
it would allow the app to get the response body if a DLC for example returns some useful information in the response body